src/docker-utils.coffee

```diff
@@ -127,9 +129,10 @@ do ->
 					supervisorTags = _.flatten(supervisorTags)
 					appTags = _.flatten(appTags)
 					locallyCreatedTags = _.flatten(locallyCreatedTags)
-					return { images, supervisorTags, appTags, locallyCreatedTags }
+					resinhupTagRegex = new RegExp("resinhup/resinhup-#{deviceType}$")
```

Comment:
I'm not particularly a fan of hardcoding this for a specific image, wouldn't this tie in with side-loading stuff? Maybe have resinhup be side-loaded and ran that way or something?